src/ol3googlemaps.js

```diff
@@ -131,6 +131,15 @@ olgm.OLGoogleMaps.prototype.getGoogleMapsActive = function() {
 
 
 /**
+ * @return {google.maps.Map}
+ * @api
+ */
+olgm.OLGoogleMaps.prototype.getMap = function() {
```

Comment:
`getMap` might be too generic.  We have 2 maps: GoogleMaps and OpenLayers.  Both have maps.

I suggest naming the method: `getGoogleMapsMap`.  That ways, it's very verbose of what it returns.